packages/fast-development-site-react/README.md

```diff
@@ -64,6 +64,25 @@ The `Site` component should be used as the wrapping component for the developmen
 
 **onUpdateDirection** - The callback provided for an update in RTL, it will execute the given callback with "rtl" or "ltr".
 
+**onUpdateTheme** - The callback provided for an update for theme, it well execute the given callback with the selected theme.
+
+#### Example callback to be passed to `onUpdateTheme`
+```jsx
+handleOnUpdateTheme = (theme: string): void {
+    this.setState({theme});
+}
+```
+**themes** - An object that passes a theme `id`,`displayName` and `background`. `background` will be the color teh component renders on top of.
```

Comment:
"An object with properties..." would be more accurate, also misspelling of "the" and I would word it more like "The `background` property will be applied to the container of the component examples on the detail and example pages.